src/tests/unit/tests/electron/platform/android/rule-information-provider.test.ts

```diff
@@ -17,44 +18,44 @@ describe('RuleInformationProvider', () => {
         expect(provider.getRuleInformation('unknown rule')).toBeNull();
     });
 
-    function validateHowToFix(ruleId: string, ruleResult: RuleResultsData): string {
+    function validateHowToFix(ruleId: string, ruleResult: RuleResultsData): InstancePropertyBag {
         const ruleInformation: RuleInformation = provider.getRuleInformation(ruleId);
-        const howToFix: string = ruleInformation.howToFix(ruleResult);
+        const howToFix = ruleInformation.howToFix(ruleResult);
 
         expect(ruleInformation).toBeTruthy();
         expect(ruleInformation.ruleId).toEqual(ruleId);
         expect(ruleInformation.ruleDescription.length).toBeGreaterThan(0);
-        expect(howToFix.length).toBeGreaterThan(0);
+        expect(howToFix).toBeTruthy();
```

Comment:
this `expect`  seems "duplicated" with the `expect(howToFix).toMatchSnapshot();` on the test themselves

Reply:
Removed